hs_core/management/commands/prevent_web_crawling.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+
+"""
+Prevent web-crawlers from indexing the site
+"""
+
+from django.core.management.base import BaseCommand
+from django.contrib.sites.models import Site
+from robots.models import Url, Rule
+
+
+class Command(BaseCommand):
+    help = "Prevent web-crawlers from indexing the site. This command is intended to be run after dev deployments."
+
+    def add_arguments(self, parser):
+
+        # a list of urls to disable
+        parser.add_argument('dissalowed_urls', nargs='*', type=str)
+
+    def handle(self, *args, **options):
+        # django-robots model definitions:
+        # https://github.com/jazzband/django-robots/blob/5.0/src/robots/models.py
+        r = Rule()
+        r.robot = "*"
+        r.crawl_delay = 5
+        r.save()  # extra save here is necessary to do r.disallowed later
+        r.sites.set(Site.objects.all())
+
+        if len(options['dissalowed_urls']) > 0:  # an array of urls to disable.
```

Comment:
I don't see this option being used anywhere.  When/wherewill this be used?

Reply:
Yeah my thought was that this management command could also be used to programmatically disable crawling of specific paths if we ever wanted to do that. We can remove the `dissalowed_urls` option if you think best

Reply:
I'll learn how to spell "disallowed" someday...